PROTOCOL.md

```diff
@@ -424,6 +424,37 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server.
+
+`|/team NUMBER|REQUEST ID`
+
+> Chooses which Pokemon to lead with.
```

Comment:
Using this format, for example `/team 4`, is only valid in Singles. Doubles/Triples/teams with Zoroark need to do something like `/team 153426`, which will determine the full order of their team.